docs/release-notes/5.7.43-31.65.md

```diff
@@ -0,0 +1,31 @@
+# Percona XtraDB Cluster 5.7.43-31.65 (2023-08-31)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and the security features your business requires to meet your customer expectations and business goals.
+
+## Release highlights
+
+Due to the [PXC-4218](https://jira.percona.com/browse/PXC-4218) bug fix, you must follow the upgrade procedure in the [Upgrading Percona XtraDB Cluster](https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/upgrade_guide.html) guide. The `Open another session and run mysql_upgrade` step is a must-have.
```

Comment:
The [PXC-4218](https://jira.percona.com/browse/PXC-4218) bug fix requires the upgrade procedure in the ... guide. Be sure that you open another session and run `mysql_upgrade` as part of the process.